selfdrive/test/process_replay/regen.py

```diff
@@ -165,22 +166,25 @@ def regen_segment(lr, frs=None, outdir=FAKEDATA):
   r = params.get("CurrentRoute", encoding='utf-8')
   return os.path.join(outdir, r + "--0")
 
-if __name__ == "__main__":
 
-  parser = argparse.ArgumentParser(description="Generate new segments from old ones")
-  parser.add_argument("--upload", action="store_true", help="Upload the new segment to the CI bucket")
-  parser.add_argument("route", type=str, help="The source route")
-  parser.add_argument("seg", type=int, help="Segment in source route")
-  args = parser.parse_args()
-
-  r = Route(args.route)
-  lr = LogReader(r.log_paths()[args.seg])
-  fr = FrameReader(r.camera_paths()[args.seg])
-  rpath = regen_segment(lr, {'roadCameraState': fr})
-  relr = os.path.relpath(rpath)
-
-  print("\n\n", "*"*30, "\n\n")
-  print("New route:", relr, "\n")
-  if args.upload:
+if __name__ == "__main__":
+  new_segments = []
```

Comment:
let's move this to a new script. I'd like to keep the current usage

Reply:
Changed back